sky/clouds/aws.py

```diff
@@ -452,10 +452,10 @@ def _get_feasible_launchable_resources(
                 region=resources.region,
                 zone=resources.zone)
             if not regions:
-                return ([], [])
+                return resources_utils.FeasibleResources([], [], None)
```

Comment:
I am leaning towards adding a `TODO` in these clouds that returns `None`, to add more hints for why a resource has failed to be found. (Our failover reasons sometimes are not helpful enough, and these additional hints should significantly help be set properly).

Reply:
Agreed, added a TODO comment for each cloud.